torchdynamo/symbolic_convert.py

```diff
@@ -721,7 +721,11 @@ def BUILD_MAP(self, inst):
             assert isinstance(k, ConstantVariable)
             result[k.value] = v
         assert len(result) == len(items) / 2
-        self.push(ConstDictVariable(result, mutable_local=MutableLocal(), **options))
+        self.push(
+            ConstDictVariable(
+                result, collections.OrderedDict, mutable_local=MutableLocal(), **options
```

Comment:
This should just be a normal dict

Reply:
I'm not fully understand why this should be normal dict. As we wrap ```result``` as a ```ConstDictVariable``` whose ```dict_type``` should be the type of ```result``` which is ```OrderedDict```?